src/components/icons/LeftArrow.tsx

```diff
@@ -0,0 +1,11 @@
+const LeftArrowIcon = () => {
+    return (
+        <svg width="23" height="16" viewBox="0 0 23 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
It's very optimal to have this icons defined this way, but I think it would be better/faster/more maintainable to just leverage some font like awesomefont, that will add value since we could reuse same style all over the app